spec/gilded_rose_spec.js

```diff
@@ -1,7 +1,206 @@
-describe("Gilded Rose", function() {
+'use strict'
 
-  it("should do something", function() {
-    update_quality();
-  });
+describe("Given Gilded Rose products", function() {
+  describe("When the day changes for normal item", function() {
 
-});
+    it("product sell_in and value properties should decrease by 1", function() {
+      update_quality();
+      expect(items[0].sell_in).toEqual(9)
+      expect(items[0].quality).toEqual(19)
+      expect(items[2].sell_in).toEqual(4)
+      expect(items[2].quality).toEqual(6)
+    })
+  })
+  describe("When update_quality on Aged Brie", function() {
+    it("Then Brie sell_in decreases", function() {
+      items[0] = new Item('Aged Brie', 2, 0)
```

Comment:
It's good your are using the `0`th item, but what happens to the performance of your tests if someone later adds 5000 items to the setup of the items array in the production code? Performace of your tests will take a hit. 

It is important to completely isolate your test data from your production data.